site/demos.md

```diff
@@ -6,19 +6,19 @@ head:
       content: Demos
   - - meta
     - name: description
-      content: Demos using Alchemy's Account Kit
+      content: Demos using Account Kit
   - - meta
     - property: og:description
-      content: Demos using Alchemy's Account Kit
+      content: Demos using Account Kit
 next:
-  text: Smart Accounts
+  text: smart accounts
```

Comment:
this is a section header right? should it be capital case?

Reply:
yes, made capital case!